src/main/java/seedu/address/model/CareFlowModelManager.java

```diff
@@ -180,4 +192,25 @@ public void updateFilteredDrugList(Predicate<Drug> predicate) {
         requireNonNull(predicate);
         filteredDrugs.setPredicate(predicate);
     }
+
+    //=========== Hospital Hotlines =========================================================
+
+    /**
+     * Set a fixed list of hospital information
+     */
+    public void setFixedHospitalList() {
+        careFlow.addHospital(new Hospital("KK Women's and Children's Hospital", "+65 62255554"));
+        careFlow.addHospital(new Hospital("Changi General Hospital", "+65 67888833"));
+        careFlow.addHospital(new Hospital("Khoo Teck Puat Hospital", "+65 65558000"));
+        careFlow.addHospital(new Hospital("Tan Tock Seng Hospital", "+65 62566011"));
```

Comment:
We may want to figure out how to store hospital data instead of hard code hospital details in ModelManager